implementations/rust/ockam/ockam_api/src/cloud/space.rs

```diff
@@ -112,6 +113,18 @@ impl Spaces for InMemoryNode {
 
     #[instrument(skip_all, fields(space_id = space_id))]
     async fn delete_space(&self, ctx: &Context, space_id: &str) -> miette::Result<()> {
+        let space_projects = self
```

Comment:
💯. Ideally, we should have some tests for this kind of behavior that is distributed across the local state and the Controller. Since we have an interface for the Controller, that should be doable.